marlowe-dashboard-client/src/MainFrame/Types.purs

```diff
@@ -120,50 +58,19 @@ data Msg
 ------------------------------------------------------------
 data Action
   = Init
-  -- pickup actions
-  | SetPickupCard (Maybe PickupCard)
-  | GenerateNewWallet
-  | PickupNewWallet
-  | LookupWallet String
-  | PickupWallet PubKey
-  -- wallet actions
-  | PutdownWallet
-  | ToggleMenu
-  | SetScreen Screen
-  | SetCard (Maybe Card)
-  | ToggleCard Card
+  -- "global" wallet data actions (can be performed in either pickup or play state)
   | SetNewWalletNickname String
-  | SetNewWalletKey PubKey
   | AddNewWallet
-  -- template actions
-  | SetTemplate Template
-  | TemplateAction Template.Action
-  -- contract actions
-  | StartContract
-  | ContractAction Contract.Action
+  -- pickup actions
+  | PickupAction Pickup.Action
+  -- play actions
+  | PlayAction Play.Action
```

Comment:
Way better 👍🏻 

No need for the group comment, as the single action is what groups them